js/tooltips.js

```diff
@@ -0,0 +1,153 @@
+import Adapt from 'core/js/adapt';
+import TooltipView from 'core/js/views/tooltipView';
```

Comment:
Please make sure the file names for tooltipView and tooltipModel are capitalised to denote that a class is returned.

Reply:
i'll amend this once you've had chance to see the diffs